modules/vre/spec/models/vre/vre_veteran_readiness_employment_claim_spec.rb

```diff
@@ -0,0 +1,190 @@
+  end
+
+  describe '#send_to_vre' do
+    subject { claim.send_to_vre(user_object) }
+
+    it 'propagates errors from send_to_lighthouse!' do
+      allow(claim).to receive(:process_attachments!).and_raise(StandardError, 'Attachment error')
+
+      expect do
+        claim.send_to_lighthouse!(user_object)
+      end.to raise_error(StandardError, 'Attachment error')
+    end
+
+    context 'when VBMS response is VBMSDownForMaintenance' do
+      before do
+        allow(OpenSSL::PKCS12).to receive(:new).and_return(double.as_null_object)
+        @vbms_client = FakeVBMS.new
+        allow(VBMS::Client).to receive(:from_env_vars).and_return(@vbms_client)
+      end
+
+      it 'calls #send_to_lighthouse!' do
+        expect(claim).to receive(:send_to_lighthouse!)
+        subject
+      end
+
+      it 'does not raise an error' do
+        allow(claim).to receive(:send_to_lighthouse!)
+        expect { subject }.not_to raise_error
+      end
+    end
+
+    context 'when VBMS upload is successful' do
+      before { expect(ClaimsApi::VBMSUploader).to receive(:new) { OpenStruct.new(upload!: {}) } }
+
+      context 'submission to VRE' do
+        before do
+          # As the PERMITTED_OFFICE_LOCATIONS constant at
+          # modules/vre/app/models/vre/constants.rb gets changed, you
+          # may need to change this mock below and maybe even move it into different 'it'
+          # blocks if you need to test different routing offices
+          expect_any_instance_of(BGS::RORoutingService).to receive(:get_regional_office_by_zip_code).and_return(
+            { regional_office: { number: '325' } }
+          )
+        end
+
+        it 'sends confirmation email' do
+          expect(claim).to receive(:send_vbms_confirmation_email).with(user_object)
+
+          claim.send_to_vre(user_object)
+        end
+      end
+
+      # We want all submission to go through with RES
+      context 'non-submission to VRE' do
+        it 'stops submission if location is not in list' do
+          expect_any_instance_of(VRE::Ch31Form).to receive(:submit)
+          claim.add_claimant_info(user_object)
+
+          claim.send_to_vre(user_object)
+        end
+      end
+    end
+
+    context 'when user has no participant ID' do
+      let(:user_object) { create(:unauthorized_evss_user) }
+
+      it 'PDF is sent to Central Mail and not VBMS' do
+        expect(claim).to receive(:process_attachments!)
+        expect(claim).to receive(:send_to_lighthouse!).with(user_object).once.and_call_original
+        expect(claim).to receive(:send_lighthouse_confirmation_email)
+        expect(claim).not_to receive(:upload_to_vbms)
+        expect(VRE::VeteranReadinessEmploymentMailer).to receive(:build).with(
+          user_object.participant_id, 'VRE.VBAPIT@va.gov', true
+        ).and_call_original
+        subject
+      end
+    end
+  end
+
+  describe '#regional_office' do
+    it 'returns an empty array' do
+      expect(claim.regional_office).to be_empty
```

Comment:
I would write this as `eq([])` because be_empty would also match an empty string or empty hash. Those may not be concerns here.